src/pages/resource/LinkPublicationPanel.tsx

```diff
@@ -1,43 +1,74 @@
 import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { RootStore } from '../../redux/reducers/rootReducer';
 
 import { Button, TextField } from '@material-ui/core';
 import LinkIcon from '@material-ui/icons/Link';
 
 import PublicationExpansionPanel from './PublicationExpansionPanel';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+import { createNewResourceFromDOI } from '../../api/resource';
+
+const StyledInputBox = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 0.3rem;
+`;
+
+const StyledTextField = styled(TextField)`
+  margin-right: 1rem;
+`;
+
+const StyledBody = styled.div`
+  width: 100%;
+`;
 
 interface LinkPublicationPanelProps {
   expanded: boolean;
   onChange: (event: React.ChangeEvent<any>, isExpanded: boolean) => void;
+  goToNextPage: any;
 }
 
-const LinkPublicationPanel: React.FC<LinkPublicationPanelProps> = ({ expanded, onChange }) => {
-  const [searchTerm, setSearchTerm] = useState('');
+const LinkPublicationPanel: React.FC<LinkPublicationPanelProps> = ({ expanded, onChange, goToNextPage }) => {
+  const [DOIUrl, setDOIUrl] = useState('');
```

Comment:
Godt eksempel på naming convention: `setDOIURL` vs `setDoiUrl`

Reply:
det er brukt lowercase på orcid, og vi kom fram til ApiBaseUrl for enum. burde vi skrevet ned navnekonvensjoner en plass?

Reply:
jeg liker konsekvent kun en stor bokstav om gangen best. Altså `doiUrl` og `setDoiUrl`